near-contract-standards/src/non_fungible_token/core/core_impl.rs

```diff
@@ -319,7 +344,7 @@ impl NonFungibleToken {
             env::panic_str("token_id must be unique");
         }
 
-        let owner_id: AccountId = token_owner_id;
+        let owner_id: AccountId = token_owner_id.clone();
```

Comment:
Seems like this one's an unnecessary clone. Everything compiles fine without it
```suggestion
        let owner_id: AccountId = token_owner_id;
```